### PR TITLE
added documentation to update the MVP docker stack

### PR DESCRIPTION
**Pull request by m1yag1**
added documentation to update the production docker stack

### CODE REVIEW

**Comment by m1yag1** [comment]
@therealmarv The `docker-compose up docs` command is more of a shortcut so someone doesn't have to start up the whole stack but I agree the ability to look at the docs should be documented. I hope to eventually have the docs living in the cluster so when someone needs to set things up they can go to cops.cnx.org/docs and move the api docs to cops.cnx.org/api

File: docs/deployment/updating_the_stack.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Setup terminal window for communicating with docker swarm manager node
+======================================================================
+
+Open another fresh terminal window. Keep it open until the end of the process.
+
+Configure docker in that terminal to use the remote host established before:
+
+.. code-block:: bash
+
+   export DOCKER_HOST="http://localhost:2377"
```
Comment (c0ldcalamari) [inline docs/deployment/updating_the_stack.rst]: `export DOCKER_HOST="http://localhost:2377"` produces error, should be
export DOCKER_HOST="localhost:2377"

**Comment by c0ldcalamari** [comment]
works when requested change is made - 
proof of working:
```
Updating service cops_prod_db (id: 4jermzk6stjmdulkzmfzisb5l)
Updating service cops_prod_frontend (id: d8t3hpy0bfobv1i57fum8usxm)
Updating service cops_prod_proxy (id: kiai1vz1296m91d9dt34xlqrp)
Updating service cops_prod_backend (id: lx7qf65m3qybsybpjth8iltv5)
```

File: docs/deployment/updating_the_stack.rst (hunk 1 above)
Comment (m1yag1) [reply docs/deployment/updating_the_stack.rst]: This has been resolved

**Pull request merged**